### PR TITLE
Explicitly initialize Point3

### PR DESCRIPTION
**Pull request by varunagrawal**
Fixes a warning in the tests. Was super easy so decided to go ahead and take care of it.

### CODE REVIEW

File: gtsam/slam/tests/testInitializePose3.cpp [hunk 1]
```diff
@@ -75,8 +75,14 @@ NonlinearFactorGraph graph2() {
   g.add(BetweenFactor<Pose3>(x0, x1, pose0.between(pose1), noiseModel::Isotropic::Precision(6, 1.0)));
   g.add(BetweenFactor<Pose3>(x1, x2, pose1.between(pose2), noiseModel::Isotropic::Precision(6, 1.0)));
   g.add(BetweenFactor<Pose3>(x2, x3, pose2.between(pose3), noiseModel::Isotropic::Precision(6, 1.0)));
-  g.add(BetweenFactor<Pose3>(x2, x0, Pose3(Rot3::Ypr(0.1,0,0.1), Point3()), noiseModel::Isotropic::Precision(6, 0.0))); // random pose, but zero information
-  g.add(BetweenFactor<Pose3>(x0, x3, Pose3(Rot3::Ypr(0.5,-0.2,0.2), Point3(10,20,30)), noiseModel::Isotropic::Precision(6, 0.0))); // random pose, but zero informatoin
+  // random pose, but zero information
+  g.add(BetweenFactor<Pose3>(x2, x0,
```
Comment (ProfFan) [inline gtsam/slam/tests/testInitializePose3.cpp:79]: format

Comment (varunagrawal) [reply gtsam/slam/tests/testInitializePose3.cpp:79]: I did format and that is what I got.

Comment (ProfFan) [reply gtsam/slam/tests/testInitializePose3.cpp:79]: Hmm. This is very strange...

Comment (varunagrawal) [reply gtsam/slam/tests/testInitializePose3.cpp]: Did a refactor so that the formatting is more consistent.

**Review by ProfFan — APPROVED** [review]
Great!

**Pull request merged**